src/main/scala/com/atomist/source/git/GitHubServices.scala

```diff
@@ -576,6 +576,12 @@ case class GitHubServices(oAuthToken: String, apiUrl: Option[String] = None)
                                             content: Option[ReactionContent] = None): Seq[Reaction] =
     listReactions(s"$api/repos/$owner/$repo/pulls/comments/$id/reactions", content)
 
+  def listIssueEvents(repo: String,
+                      owner: String,
+                      number: Int,
+                      params: Map[String, String] = Map("per_page" -> "100")): Seq[IssueEvent] =
+    paginateResults[IssueEvent](s"$api/repos/$owner/$repo/issues/$number/events", params)
```

Comment:
`paginateResults` and `paginateSearchResults` don't retry. I'm not sure if I should make it. What do you think?